app/controllers/message_handler_controller.rb

```diff
@@ -7,22 +8,15 @@ def handle
     # regardless of whether we're doing a `bundle exec` call or we're
     # getting the messages fed to us by SQS via POST to this HTTP endpoint
     # as is the standard for the Worker Tier in Elastic Beanstalk.
-    resp = QueueListener.new.perform(nil, params)
+    response = QueueListener.new.perform(nil, params)
 
     # Tell SQS everything went hunky dory and we can delete the message.
-    send_status_ok
+    render json: response, status: 200
```

Comment:
It's for integration testing - I run expectations against the response. So long as the queue gets back a 200, we're good.